rebasehelper/cli.py

```diff
@@ -157,7 +157,11 @@ def run():
             logger.info('\nInterrupted by user')
         except RebaseHelperError as e:
             logger.error('\n%s', six.text_type(e))
+            logger.debug('\n', exc_info=1)
```

Comment:
with RebaseHelperError I don't think we need to log the traceback. This type of exception is used as a way to express expected failures.